config-mancer/helpers.js

```
@@ -0,0 +1,32 @@
+        '@': 'string',
+        'fileName': 'string',
+    }
+
+    static configMancerFactory(obj) {
+        const pathName = (obj['@baseDir'] && !path.isAbsolute(obj.fileName)) ?
+            path.join(obj['@baseDir'], obj.fileName) :
+            obj.fileName;
+        const text = readFileSync(pathName, 'utf8');
+        return text;
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)